R/src.R

```diff
@@ -6,6 +6,98 @@
 
 ############## Parsers ##############
 
+#' Parse file dependencies from Scitool's Understand
+#'
+#' @param project_path path to the project folder to analyze
+#' @param language the language of the project (language must be supported by Understand)
+#' @param output_dir path to output directory (formatted output_path/)
+#' @param parse_type Type of dependencies to generate into xml (either "file" or "class")
+#' @export
+#' @family parsers
+understand_parse_dependencies <- function(project_path,language,output_dir="../tmp/", parse_type){
+  # Before running, check if parse_type is correct
+  validInput <- function(input) {
+    is.character(input) && length(input) == 1 && input %in% c("file", "class")
+  }
+  if (!validInput(parse_type)) {
+    stop("Error: Invalid parse_type provided. Please input either \"file\" or \"class\"")
+  }
+
+  # Use Understand to parse the code folder.
+  # Create the variables used in command lines
+  project_path <- paste0("\"", project_path, "\"")
+  db_dir <- paste0(output_dir, "/Understand.und")
+  xml_dir <- paste0(db_dir, "/", parse_type, "Dependencies.xml")
+  command <- "und"
+  args <- c("create", "-db", db_dir, "-languages", language)
```

Comment:
This may need to be a separate function. It does not make sense for scitools to re-create its database every time. What you may create multiple times is the XML file out of the DB. Did you try running the tool manually to get the Class and the File XML out of it? Did creating one DB suffice to them use the other commands to extract the XML files? 

If anything, I worry that having the DB here will mean the second function call will fail. 

@crepesAlot note this is a new tool that has to appear in the project configuration file. Did you coordinate?